README.md

```diff
@@ -297,7 +297,7 @@ ignore = E501,W503,E203
 ```
 
 You'll find *Black*'s own .flake8 config file is configured like this.
-If you're curious about the reasoning behind B950, 
+If you're curious about the reasoning behind B950,
```

Comment:
Sorry, VSCode automatically removed the trailing whitespace here and below. If you want a minimal diff, I'll add a commit reverting this.

Reply:
It's fine to keep.